### DISCUSSION TITLE
Emulador não recebe as informações do app

### DISCUSSION THREAD

**Discussion by JoaoPedroDosSantosAlves** [Flutterando/forum#204] [Perguntas e Respostas]
Sou iniciante, meu emulador parou de receber as informações do codigo, ao rodar pelo VSCODE nem mostra o app criado. Quais possiveis erros, espero que me ajudem

**Comment by KaioBorgesDev** [comment]
Olá, conseguiu resolver ? Caso não, consegue dizer qual emulador, linguagem, biblioteca e.t.c para nortear melhor ?

**Reply by JoaoPedroDosSantosAlves** [reply]
Opa, consegui sim man, obrigado!
